asset_pipeline/pipeline.py

```diff
@@ -132,19 +131,30 @@ def __init__(self,
         self.ssl = config['ssl']
         if self.ssl:
             self.protocol = self.protocol + 's'
+        # authenticated client
+        self.client = get_client_for_config(config)
         logger.info('Running based on %s' % self)
 
     def validate_configuration(self, config):
         # make sure hostname and port are set
+        NOT_PROVIDED_ERROR_MSG = '{0} needs to be provided in order to connect the pipeline to the hub'
         if 'host' not in config:
-            logger.error("Hostname needs to be provided in order to connect the pipeline to the holocloud®")
+            logger.error(NOT_PROVIDED_ERROR_MSG.format('Hostname'))
             return False
         if 'port' not in config:
-            logger.error("Port needs to be provided in order to connect the pipeline to the holocloud®")
+            logger.error(NOT_PROVIDED_ERROR_MSG.format('Port'))
             return False
         # if we got here, everything's fine
         return True
 
+    def authenticate_headers(self, headers):
```

Comment:
headers cannot be authenticated. Authorization information can be added to the headers or the request can be authenticated.